common/xlStackWalker.h

```diff
@@ -0,0 +1,92 @@
+#pragma once
```

Comment:
This file is a better version of MSWStackWalk.h (which was deleted). This file has some fixes for handling exception cases and uses wxWidgets where possible.